.devcontainer/Dockerfile

```diff
@@ -24,9 +24,12 @@ ARG OCAML_VERSION
 USER root
 RUN apt-get install -y m4
 USER opam
-RUN opam update && opam install ocaml-lsp-server
+RUN opam update && opam install ocaml-lsp-server ocamlformat
 RUN opam user-setup install
 
+# Install utop
+RUN sudo apt-get install -y utop
```

Comment:
Kako to, da `utop`a ne namestite z OPAMom?

Reply:
Pri namestitvi z `opam`om je prišlo do napake.

Reply:
Ja, razlog je verjetno v tem, da nekateri `opam`ovi paketi potrebujejo zunanje knjižnice, ki niso nameščene, `apt-get` pa jih zna namestiti sam. V tem primeru je boljša rešitev ukaz `opam depext` (v starih različicah OPAMa gre za dodaten plugin, v novejših je že vgrajen), ki sam poišče in po potrebi namesti ustrezne zunanje knjižnice.